src/app/cluster/list/cluster/component.ts

```diff
@@ -311,4 +325,27 @@ export class ClusterListComponent implements OnInit, OnChanges, OnDestroy {
         this.clusterTemplates = templates;
       });
   }
+
+  private _loadMachineDeployments(cluster: Cluster): void {
+    if (Health.allHealthy(this.health[cluster.id]) && !cluster.deletionTimestamp) {
+      this._machineDeploymentService
+        .list(cluster.id, this._selectedProject.id)
+        .pipe(takeUntil(this._unsubscribe))
+        .subscribe({
+          next: machineDeployments => {
+            const mdStatus: MachineDeploymentStatus = {
+              replicas: 0,
+              availableReplicas: 0,
+              updatedReplicas: 0,
+            } as MachineDeploymentStatus;
+            machineDeployments.forEach(machineDeployment => {
+              mdStatus.replicas += machineDeployment.status?.replicas || 0;
```

Comment:
I think if i know correctly?  -> 

spec ={} shows desired state
status ={} shows current state

here replicase are reading from status -> which i think currently working fine because because BE is returning same `replica` count as of spec one but not sure because same check is used in html and may be it cause some issue? wdyt?

Reply:
im little confused regarding what do we need to show on the cluster list, replicas or MD i think we need to show number of MD in the cluster list but right now it shows the number of replica e.g if i have one md with two replicas it supose to show in the cluster list 1/1 not 2/2 right ?

Reply:
In the issue it is specifically mentioned to show number of replicas.